gulpfile.js

```diff
@@ -161,13 +212,15 @@ gulp.task('build', gulp.series(
 	// webpackServices,
 	webpackComponents,
 	// webpackReactComponents,
-	buildSass
+	buildSass,
+	angularBuild
```

Comment:
Is build order important for these tasks? Do the angular components have SASS to build?

Reply:
Angular build compiles its own Sass, our build was trying (and failing) to compile some of the default Sass (I think blowing up on some test). We now ignore Sass in angular components and let ng build handle it and generate its own files...